telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,26 @@
 import logging
 import time
 import warnings
+import datetime
+from numbers import Number
 from threading import Thread, Lock, Event
 from queue import PriorityQueue, Empty
 
 
+class Days(object):
+    mon = 0
```

Comment:
Since we switched to a class with constants, the correct naming convention would be `UPPER_CASE` if I'm not mistaken.